src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -41,9 +45,12 @@ export default class DetailsBody extends React.Component<Props> {
     const discoverQuery: NewQuery = {
       id: undefined,
       name: (incident && incident.title) || '',
-      fields: ['title', 'user', 'last_seen'],
+      fields: ['issue.id', 'count(id)', 'count_unique(user.id)'],
```

Comment:
You can use the `issue` field now to get a shortid link instead of the autoincement.